### DISCUSSION TITLE
人工智慧取代工作?

### DISCUSSION THREAD

**Discussion by hardtagottaname** [1101-GS4719-C/Python-Community-GS4719-C#1306] [General]
ChatGpt提供人們便利的學習環境，藉由輸入幾行指令，ChatGpt便會剖析問題，並鉅細靡遺的解釋答案。
然而，正是因為ChatGpt過於便利，越來越多人開始反思，現在人們所做的工作會不會有一天被人工智慧取代。
我認為具備以下幾種性質的工作不會被人工智慧取代:
(1)需要與人交流:
    社交是人與人之間必備的技能，人工智慧很難根據人的表情、語氣、用詞猜測人類所想所思(人類很常所說的和所想的不盡相同)。
(2)需要創新:
    人工智慧說穿了不過是將舊有的大數據做分析，將所有的舊有資料進行各種排列組合。然而，人類的大腦卻能無中生有，這是人工智慧所做不到的。
因此，藉由以上兩點，我認為並非所有工作都會被人工智慧取代。

**Comment by YoYooHuang** [comment]
我認為chatgpt就是匯集所有專業知識的資料庫，任何人都能夠從中得知一些專業的相關資訊。我覺得我們必須具備的是資訊正確性的判斷能力，當我們得到資訊時，不是全然接受，而是要仔細應證。最明顯的例子是，我曾經用他來問過作業怎麼寫，但他的回答跟答案明顯有出入，可是讀的題目明明是一模一樣的。為了讓他解出正確答案，我嘗試一步一步引導他，要用哪個公式，步驟是什麼，他才成功解出正確答案。

**Comment by KevinCheng42** [comment]
我同意作者的觀點，創新或需要創意發想的工作比較難被人工智慧取代，因為目前的技術還無法讓人工智慧創造新事物或想法。然而，大數據分析可以用於分析趨勢，這也使得許多大型服飾品牌透過此技術來分析人們的喜好和審美觀，進而推出相似概念的商品。

**Comment by Jack-1D** [comment]
我認為娛樂業也不太會被取代，不論是歌手、演員，甚至是運動員。人們還是比較喜歡看到這些領域中的人互相較量，激盪出好成績或好作品。但這也僅止於現階段，未來有沒有可能機器人對決成為焦點，也很難說。

**Comment by JessyCheng2003** [comment]
我認為創新也是可以被取代的，昨天看到一個人分享，他要幫防曬產品做廣告設計圖，於是他先問了Chatgpt圖片的建議，再將chatgpt給的圖片建議複製到"midjorney"上，midjorney就自動幫她生成出廣告圖，整個過程只花了30分鐘，所以我覺得在創新方面還是可以被取代。

**Comment by lyien1021** [comment]
我記得老師有說過，未來的工作不會被AI取代，而是會被懂AI的人取代，我覺得蠻有道理，現在修AI相關課程的人越來越多，如果完全不了解AI的話很容易就被淘汰。

**Comment by pudding0803** [comment]
我認為還有一個能力也不容易被 AI 取代，也就是「結合多個領域」的能力。如果某個能力需要運用跨領域的知識的話，除了現階段很難被其他人取代之外，未來應該也不容易被 AI 取代。總而言之，平時就多多學習、積攢自己的能力吧😇

**Comment by youtinglin** [comment]
我認為與其擔心未來的工作會被AI取代，不如說找到自身的優勢是無法被取代的，思考自己的價值，因社會的趨勢變化逐漸加速，除了適應力的提升，還有就是不論何時都要以開放的心胸去接納新的改變，了解新知識，隨時學習，去調整自己每個階段要努力的方向。

**Comment by Alexgns** [comment]
雖然人工智慧在某些領域取得了重大進展，但它更多地被視為一個協助工具，而非完全取代人類工作的替代品。人類具備複雜的情感、創造力和人際互動能力，這些特質使得我們在很多工作領域仍然不可或缺。相反，人工智慧的應用能夠提高工作效率，讓人們專注於更具挑戰性和創造性的任務。

**Comment by xiang305** [comment]
我認為管理層面的職位也不會被取代，畢竟人類應該不太願意被人工智慧管。而且萬一管理流程出了什麼差錯，在追究責任時，我們還是需要讓人類來為這件事負起責任或給出實質補償

**Comment by ashleyy-huang** [comment]
我認為最不會被人工智慧取代的就是人類的思想與創意，這些東西是活的，但人工智慧所使用的是知識庫和一筆筆資料，難以產生出資料範圍外的其他知識，與其說人工智慧會取代工作，我認為他只能做到輔助的效果，提升人類的工作效率
